docs/how-to-run.md

```diff
@@ -0,0 +1,168 @@
+# Running lookout
+
+The recommended way to locally try **source{d} lookout** is using `docker-compose` as described in [quickstart documentation](../README.md#quickstart).
+
+In some situations, you might want to run all its dependencies and components separately, or you can even want to [run it in a distributed mode](#running-lookout-in-distributed-mode). Both ways to run lookout would require to follow these general steps:
+
+1. [Run lookout dependencies](#dependencies),
+1. [Run the desired analyzers](#run-the-analyzers),
+1. [Configure lookout](#configure-lookout),
+1. [Run lookout](#run-lookout).
+
+In other situations you just want to locally test an analyzer without accessing GitHub at all; for that situations, you might want to read [`lookout-sdk` binary documentation](lookout-sdk.md).
+
+## Running lookout in Distributed Mode
+
+lookout can be also run in a distributed fashion using a [RabbitMQ](https://www.rabbitmq.com/) queue to coordinate a _watcher_ and several _workers_.
+
+- The _watcher_ process monitors GitHub pull requests and enqueues jobs for new events.
+- The running _workers_ dequeue jobs as they become available calling the registered analyzers and posting its returned comments.
+
+The general steps to run source{d} lookout in distributed mode are the same as said above.
+
+
+# Dependencies
+
+**lookout** needs a running instance of:
+
+* [bblfshd](https://github.com/bblfsh/bblfshd) to parse files into [UAST](https://doc.bblf.sh/uast/uast-specification.html).
+* [PostgreSQL](https://www.postgresql.org) to internally store `PushEvent`, `ReviewEvent`, `ReviewTarget` and `Comment`.
+* (**optional**) [RabbitMQ](https://www.rabbitmq.com) to coordinate a _watcher_ and several _workers_ (when running lookout in a distributed way).
+
+You can run them manually or with `docker-compose`.
+
+```bash
+$ docker-compose up -d --force-recreate bblfsh postgres
+```
+
+In case you want to run it in a distributed way, you will also need RabbitMQ, so you can run instead:
+
+```bash
+$ docker-compose -f docker-compose.yaml -f docker-compose-rabbitmq.yml up -d --force-recreate bblfsh postgres rabbitmq
```

Comment:
Do we really want to provide a docker-compose file with rabbitmq? I see its value for us, as developers. But for someone looking to deploy lookout in a distributed way, will they care about running the distributed scenario in their own laptop?

On the other hand taking a look at the compose file can be helpful to understand how to deploy each component manually. :thinking:

Reply:
I think it worth having it in a separated file, that can be used if needed, and is consistent with how different dependencies are run.